### PR TITLE
Add hint for rescan of external table [#115361337]

### PR DESCRIPTION
**Pull request by xinzweb**
@yydzero @vraghavan78 @addisonhuddy @hsyuan

### CODE REVIEW

**Comment by hsyuan** [comment]
+1

**Comment by oarap** [comment]
+1

File: src/backend/access/external/url.c [hunk 1]
```diff
@@ -2492,7 +2492,11 @@ url_rewind(URL_FILE *file, const char *relname)
 			break;
 
 		case CFTYPE_EXEC:
-			elog(ERROR, "Rescan is not supported for web external table: %s", relname);
+			{
+				elog(ERROR, "Rescan is not supported for web external table: %s. "
+						"Please use 'set optimizer=on' as a work around "
+						"and 'set optimizer_enable_master_only=on' if accessing catalog tables.", relname);
+			}
 			break;
```
Comment (vraghavan78) [inline src/backend/access/external/url.c]: what if the OSS developer did not run it with enable-orca. PM's ask it to have us say install with enable-orca.

File: src/backend/access/external/url.c [hunk 2]
```diff
@@ -2492,7 +2492,11 @@ url_rewind(URL_FILE *file, const char *relname)
...
 
 		case CFTYPE_EXEC:
-			elog(ERROR, "Rescan is not supported for web external table: %s", relname);
+			{
+				elog(ERROR, "Rescan is not supported for web external table: %s. "
+						"Please use 'set optimizer=on' as a work around "
```
Comment (danielgustafsson) [inline src/backend/access/external/url.c]: In this sentence I think the noun form "workaround" rather then verb form is more suitable.

**Pull request merged**